recipes/code-server/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+VERSION=${PKG_VERSION//_/-}
+
+vscodeVersion=$(python -c "print('${VERSION}'.split('-vsc')[1])")
+codeServerVersion=$(python -c "print('${VERSION}'.split('-vsc')[0])")
+
+yarn
+MINIFY=true yarn build "${vscodeVersion}" "${codeServerVersion}"
+yarn binary "${vscodeVersion}" "${codeServerVersion}"
+mkdir -p ${PREFIX}/bin/
+mv binaries/code-${VERSION}-*-x86_64 ${PREFIX}/bin/
```

Comment:
It has to be `code-server${VERSION}`

Reply:
```
[binary] Binary: $SRC_DIR/binaries/code-server2.1692-vsc1.39.2-darwin-x86_64
Done in 8.27s.
```
So it has to be `code-server${VERSION}` instead of `code${VERSION}` or `code-${VERSION}`

Reply:
🤦‍♂ Oh, sorry. Was too early for me.